docs/kubernetes/user-guides/snapshots.md

```diff
@@ -19,33 +23,89 @@ $ ./deploy/kubernetes/deploy-driver.sh
 
 ### Snapshot Example
 
-1. Create example Default Snapshot Class
+**Create Storage Class:**
+
+If you haven't created a `StorageClass` yet, create one first:
+
+```console
+kubectl apply -f ./examples/kubernetes/demo-zonal-sc.yaml
 ```
-$ kubectl create -f ./examples/kubernetes/demo-defaultsnapshotclass.yaml
+
+**Create  Default VolumeSnapshotClass:**
+
+```console
+kubectl create -f ./examples/kubernetes/snapshot/default_volumesnapshotclass.yaml
```

Comment:
nit: I believe we use `-` throughout the rest of the repo instead of `_`. Could you change the underscores to hyphens?

Reply:
Done.